client/blocks/daily-post-button/test/index.jsx

```diff
@@ -1,43 +1,40 @@
 /** @format */
+jest.mock( 'reader/stats', () => ( {
+	pageViewForPost: () => {},
+	recordAction: () => {},
+	recordGaEvent: () => {},
+	recordTrackForPost: () => {},
+} ) );
+jest.mock( 'lib/analytics', () => ( {
+	mc: {
+		bumpStat: () => {}
+	}
+} ) );
+jest.mock( 'page', () => require( 'sinon' ).spy() );
+jest.mock( 'components/sites-popover', () => {
```

Comment:
It turns out this component loads `lib/user` which depends on DOM. I updated mocks to reflect this.